Controller/ResourceController.php

```diff
@@ -123,17 +123,39 @@ protected function findOrThrowNotFound(ResourceInterface $resource, $id)
     protected function getCollectionData(ResourceInterface $resource, Request $request)
     {
         $page = (int) $request->get('page', 1);
-
+        $order = [];
         $filters = [];
-        foreach ($resource->getFilters() as $resourceFilter) {
+        $resourceFilters = $resource->getFilters();
+        $resourceOrders = $resource->getOrder();
+
+        foreach ($resourceFilters as $resourceFilter) {
             if (null !== $value = $request->get($resourceFilter['name'])) {
                 $resourceFilter['value'] = $value;
                 $filters[] = $resourceFilter;
             }
         }
 
+        // Add order filters
+        $requestOrderFilters = (null !== $request->get('order'))?
+            $request->get('order'):
+            [];
+        foreach ($requestOrderFilters as $key => $value) {
+            foreach ($resourceOrders as $resourceOrder) {
+                if ($resourceOrder['name'] === $key
+                    && ('asc' === strtolower($value) || 'desc' === strtolower($value))) {
```

Comment:
As mentioned in a previous comment, the test for `ASC` and `DESC` should be done in `DoctrineDataProvider` and not here. Some `DataProvider` can support other type of sort (pertinence for an ElasticSearchProvider for instance).